BuildCommands.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Reflection;
+
+static class BuildCommand
+{
+
+    private static void SyncSolution() {
+        Type
+            .GetType("Packages.Rider.Editor.RiderScriptEditor, Unity.Rider.Editor")
```

Comment:
So I got it from here becase the SyncSolution methods provided by unity were not working for me. Maybe I should add a link to this and the reasons mentioned here? https://github.com/sbergen/Responsible/blob/main/ResponsibleUnity/Assets/EditorSetup/ContinuousIntegration.cs#L53-L63

Reply:
Ah I see. Maybe just slap the attribute they have on it.
```[SuppressMessage(
            "ReSharper",
            "PossibleNullReferenceException",
            Justification = "Having to use reflection because of internal types.")]```